src/extension/kernel.ts

```diff
@@ -209,6 +210,26 @@ export class Kernel implements Disposable {
       return window.showInformationMessage(message.output as string)
     } else if (message.type === ClientMessages.openLink) {
       return env.openExternal(Uri.parse(message.output))
+    } else if (message.type === ClientMessages.githubWorkflowDispatch) {
+      const { itFailed, reason, workflowRun } = await deployWorkflow(message.output)
+      postClientMessage(this.messaging, ClientMessages.githubWorkflowDeploy, {
+        itFailed,
+        reason,
+        workflowRun
+      })
+
+      if (workflowRun) {
+        await checkWorkflowRunStatus({
+          owner: message.output.owner,
+          repo: message.output.repo,
+          run_id: Number(workflowRun.id),
+          onStatusUpdate: (workflowRun) => {
+            postClientMessage(this.messaging, ClientMessages.githubWorkflowStatusUpdate, {
+              workflowRun
+            })
+          }
+        })
+      }
```

Comment:
Maybe we can return early here and safe indentation
```suggestion
      if (!workflowRun) {
        return
      }
      await checkWorkflowRunStatus({
        owner: message.output.owner,
        repo: message.output.repo,
        run_id: Number(workflowRun.id),
        onStatusUpdate: (workflowRun) => {
          postClientMessage(this.messaging, ClientMessages.githubWorkflowStatusUpdate, {
            workflowRun
          })
        }
      })
```

Reply:
Fixed